Feat/convert task

MaxPostema opened the pull request.

mswertz commented:
Hi, there in https://github.com/molgenis/molgenis-emx2/pull/1520 I had to already migrate task and subtask.
So you can either take those or simply skip them.

MaxPostema commented:
> Hi, there in #1537 I had to already migrate task and subtask. So you can either take those or simply skip them.

I suggest using this variant of the Task component as we removed a 'memory leak' fixed it cluttering up the error log and cleaned up some of the used code style.

mswertz commented on apps/molgenis-components/src/components/task/Task.vue:
```diff
@@ -0,0 +1,83 @@
+<template>
+  <div>
+    <demo-item>
+      <Task taskId="6956bf6d-3f78-4798-873f-dd5382ac0e24"/>
```
I think we should add a stringinput where you can enter the taskID.
What you also can do is to only demo as part of taskList

mswertz commented:
PR looks good, I think only task demo should be parameterized (with stringinput to provide a taskid) otherwise it will never really work.
